### PR TITLE
Add note about inclusion in the VOID

### PR DESCRIPTION
**Pull request by lawrencejones**
The VOID has included all the post-mortems from this list, including
many others.

Anyone visiting this list would probably benefit from knowing about the
VOID, so add a note to help them find it.

### CODE REVIEW

**Comment by danrl** [comment]
I was asked to approve this PR. Although our names are similar and both Dan and I work in the same industry, I don't have any power here.

FWIW, without power or authority, I approve this PR. :) 

Also: Hi @danluu 👋

**Comment by lawrencejones** [comment]
:joy: this interaction confused me much more than it should have!

Screenshot 2022-03-14 at 15 45 09

File: README.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # A List of Post-mortems!
 
+> This list of post-mortems has been included in the Verica Open Incident
+> Database (https://www.thevoid.community/) if you wish to browse them there.
```
Comment (icco) [inline README.md:4]: We have a list of other postmortems at the bottom of this document. Please move this comment to there.

**Review by icco — CHANGES_REQUESTED** [review]

**Comment by icco** [comment]
No update, closing.

**Pull request closed without merging**